### ISSUE TITLE
Demo might need some fixes and update

### ISSUE THREAD

**Issue by XiaochaoYan** [AndresTraks/BulletSharp#65]
Compiling on a .Net8 and Win11 PC.

Some of the errors.
1. on file "BasicDemo.cs" line 74 
CS0019	Operator '*' cannot be applied to operands of type 'Matrix' and 'Matrix4'	BasicDemo	C:\Users\Leon_\Desktop\BulletSharp-master\demos\OpenTK\BasicDemo\BasicDemo.cs	77		

2. on file "Physics.cs" line 32
CS0029	Cannot implicitly convert type 'OpenTK.Vector3' to 'BulletSharp.Math.Vector3'	BasicDemo	C:\Users\Leon_\Desktop\BulletSharp-master\demos\OpenTK\BasicDemo\Physics.cs	32		

Basically some data type conversion/parsing issues.
